standard_rules/unusual_login.yml

```diff
@@ -0,0 +1,58 @@
+AnalysisType: rule
+Filename: unusual_login.py
+RuleID: Standard.UnusualLogin
+DisplayName: Unusual Login
+# This rule is disabled by default because it makes API calls out to the internet. At high volumes
+# of activity, this could get throttled unless you buy a subscription to ipinfo.
+Enabled: false
+LogTypes:
```

Comment:
I think `AWS.CloudTrail` also had the required data model fields for this detection.